specification/metrics/semantic_conventions/runtime-environment-metrics.md

```diff
@@ -398,5 +402,83 @@ This metric is obtained from [`BufferPoolMXBean#getCount()`](https://docs.oracle
 **[1]:** Pool names are generally obtained via [BufferPoolMXBean#getName()](https://docs.oracle.com/en/java/javase/11/docs/api/java.management/java/lang/management/BufferPoolMXBean.html#getName()).
 <!-- endsemconv -->
 
+### Metric: `process.runtime.jvm.cpu.monitor.duration`
+
+This metric is [recommended][MetricRecommended]. Only available with JDK 17+.
+This metric is obtained from [`jdk.JavaMonitorWait`](https://sap.github.io/SapMachine/jfrevents/21.html#javamonitorwait) and [`jdk.JavaMonitorEnter`](https://sap.github.io/SapMachine/jfrevents/21.html#javamonitorenter) JFR events.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-advice)
+of `[]` (single bucket histogram capturing count, sum, min, max).
+
+<!-- semconv metric.process.runtime.jvm.cpu.monitor.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `process.runtime.jvm.cpu.monitor.duration` | Histogram | `s` | Time monitor was used by a thread. Only available in JDK 17+. |
+<!-- endsemconv -->
+
+<!-- semconv metric.process.runtime.jvm.cpu.monitor.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `class` | string | Class of the monitor. | `java.lang.Object` | Opt-In |
+| `state` | string | Action taken at monitor. | `blocked`; `wait` | Recommended |
```

Comment:
Should note that we we may need to add a namespace to this dependent on the result of #51.